fastlane/swift/Fastlane.swift

```diff
@@ -230,15 +230,17 @@ public func appetize(apiHost: String = "api.appetize.io",
                      platform: String = "ios",
                      path: String? = nil,
                      publicKey: String? = nil,
-                     note: String? = nil)
+                     note: String? = nil,
+                     timeout: Integer? = nil)
```

Comment:
Can't comment on the exact line, but we could update the docs for this new parameter too. Same with the `buildAndUploadToAppetize` function below.